Source/Core/Common/GekkoDisassembler.cpp

```diff
@@ -376,7 +376,7 @@ std::string GekkoDisassembler::imm(u32 in, int uimm, int type, bool hex)
 
   case 1:
     if (hex)
-      return fmt::format("{}, {}, 0x{:.4X}", regnames[PPCGETA(in)], regnames[PPCGETD(in)], i);
+      return fmt::format("{}, {}, 0x{:04X}", regnames[PPCGETA(in)], regnames[PPCGETD(in)], i);
```

Comment:
Thanks for catching this one; I missed it [in my comment earlier](https://github.com/dolphin-emu/dolphin/pull/8177#discussion_r294068074).